main.go

```diff
@@ -87,7 +90,14 @@ func obfuscate(keepTests, outGopath bool, encKey, pkgName, outPath string) bool
 	if !outGopath {
 		ctx := build.Default
 		newPkg := encryptComponents(pkgName, enc)
-		cmd := exec.Command("go", "build", `-ldflags=-s -w -extldflags "-static"`, "-o", outPath, newPkg)
+		var ldflags string
+		if winHide == true {
+			ldflags = `-ldflags=-s -w -extldflags "-static" -H=windowsgui`
```

Comment:
There's some duplicated code here (since some of the flags are repeated in the else clause). Could you change to

```go
ldflags := `-ldflags=-s -w -extldflags "-static"`
if winHide {
    ldflags += " -H=windowsgui"
}
```

Reply:
Ok, i made the necessary changes